salt/config/__init__.py

```diff
@@ -1007,7 +999,7 @@ def _gather_buffer_space():
     'gitfs_global_lock': True,
     'gitfs_ssl_verify': True,
     'gitfs_saltenv': [],
-    'hash_type': 'sha256',
+    'hash_type': 'md5',
```

Comment:
We're definitely not going to change this back. See larger discussion here: https://github.com/saltstack/salt/pull/35341